src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java

```diff
@@ -2896,4 +2900,85 @@ void testIssueWithReactiveTerms() {
+        assertEquals(1.087, postContingencyResult1.getNetworkResult().getBusResult("b3").getV(), DELTA_V); // tr34 is controlling voltage of b3
+        assertEquals(0.8755940, postContingencyResult1.getNetworkResult().getBranchResult("tr34").getExtension(OlfBranchResult.class).getContinuousR1(), DELTA_RHO);
+
+        // Verify same results are obtained with l23 initially disconnected and an operator strategy reconnecting it
+        network.getLine("l23").disconnect();
+        List<Action> actions = List.of(new TerminalsConnectionAction("close_l23", "l23", false),
+                                       new TerminalsConnectionAction("close_l01", "l01", false));
+        List<OperatorStrategy> operatorStrategies = List.of(new OperatorStrategy("strategy1", ContingencyContext.specificContingency("contingency1"), new TrueCondition(), List.of("close_l23", "close_l01")));
+        result = runSecurityAnalysis(network, List.of(new Contingency("contingency1", new BranchContingency("l01"))),
+                monitors, securityAnalysisParameters, operatorStrategies, actions, Reporter.NO_OP);
+
+        // pre-contingency verification
+        preContingencyResult = result.getPreContingencyResult();
+        assertEquals(LoadFlowResult.ComponentResult.Status.CONVERGED, preContingencyResult.getStatus());
+        assertEquals(1.000, preContingencyResult.getNetworkResult().getBusResult("b1").getV(), DELTA_V); // g0 is controlling voltage of b1
+        assertEquals(1.087, preContingencyResult.getNetworkResult().getBusResult("b3").getV(), DELTA_V); // tr34 is controlling voltage of b3
+
+        // post-contingency verification: l01 contingency prevents g0 to hold voltage at b1, tr34 holds voltage at b3
+        postContingencyResult0 = getPostContingencyResult(result, "contingency1");
+        assertEquals(PostContingencyComputationStatus.CONVERGED, postContingencyResult0.getStatus());
+        assertEquals(1.043, postContingencyResult0.getNetworkResult().getBusResult("b1").getV(), DELTA_V); // voltage is not held at b1
+        assertEquals(1.111, postContingencyResult0.getNetworkResult().getBusResult("b3").getV(), DELTA_V); // tr34 is controlling voltage of b3
+        assertEquals(0.9066748, postContingencyResult0.getNetworkResult().getBranchResult("tr34").getExtension(OlfBranchResult.class).getContinuousR1(), DELTA_RHO);
+
+        // post-operator strategy verification: applied action merges the zero impedance networks, target voltage of tr34 is applied.
+        OperatorStrategyResult operatorStrategyResult = getOperatorStrategyResult(result, "strategy1");
+        assertEquals(PostContingencyComputationStatus.CONVERGED, operatorStrategyResult.getStatus());
+        assertEquals(1.1, operatorStrategyResult.getNetworkResult().getBusResult("b3").getV(), DELTA_V); //  g0 is controlling voltage of b3
+        assertEquals(1.1, operatorStrategyResult.getNetworkResult().getBusResult("b1").getV(), DELTA_V); // with tr34 targetV
```

Comment:
we have an issue here, g0 cannot regulate voltage - g0 has been 'indirectly outaged" by `l01` line contingency

Reply:
same as https://github.com/powsybl/powsybl-open-loadflow/pull/955#discussion_r1511017638
apologies / resolving